README.md

```diff
@@ -153,6 +153,22 @@ It is also possible to use `[@polyprinter]`. The difference is that for a type `
 
 The function `fprintf` is locally defined in the printer.
 
+By default all constructors are printed with prefix which is dot-separated filename and module path. For example
+``` ocaml
+(* A.ml *)
+module X = struct type t = C [@@deriving show] end
+```
+
+This code will create printers which return the string `A.X.C` where `A` is a filename, `X` is a module path and `C` is a constructor name. To skip all module paths the one needs to derive show with `nofullpath` option (which equals to `false` by default)
+
+``` ocaml
+# module X = struct type t = C [@@deriving show { nofullpath = true }] end;;
+...
+# X.show (X.C);;
+- : Ppx_deriving_runtime.string = "C"
+```
```

Comment:
The first example shows just the code while this one is in the context of a toplevel session. Maybe they should both be in the toplevel or both be just code?